data/categories.txt

```diff
@@ -11808,6 +11808,7 @@ amenity-charging_station-bicycle|@charging_station
 en:4Bicycle Charging|4Bike Charging
 ru:4Велозарядка
 tr:4Bisiklet Şarj Etme
+de: Rad laden|Fahrrad laden
```

Comment:
1. The space after : is not needed.

2. These are translations from types_strings.txt , are they correct? What do users usually type in the search to find a bicycle charging station in your language? @organicmaps/translations
```
    en = Bicycle Charging Station
    af = Oplaaistasie vir fietse
    ar = محطة شحن الدراجات
    az = Velosiped doldurma stansiyası
    be = Станцыя для зарадкі ровараў
    bg = Станция за зареждане на велосипеди
    ca = Estació de recàrrega de bicicletes
    cs = Nabíjecí stanice pro jízdní kola
    da = Cykel ladestation
    de = Fahrrad Ladestation
    el = Σταθμός φόρτισης ποδηλάτων
    es = Estación de carga de bicicletas
    et = Jalgratta laadimisjaam
    eu = Bizikletak kargatzeko geltokia
    fa = ایستگاه شارژ دوچرخه
    fi = Polkupyörän latausasema
    fr = Borne de recharge pour vélo
    he = תחנת טעינה לאופניים
    hu = Kerékpártöltő állomás
    id = Stasiun Pengisian Sepeda
    it = Stazione di ricarica per biciclette
    ja = 自転車充電ステーション
    ko = 자전거 충전소
    mr = सायकल चार्जिंग स्टेशन
    nb = Ladestasjon for sykkel
    nl = Oplaadstation voor fietsen
    pl = Stacja ładowania rowerów
    pt = Estação de carregamento de bicicletas
    pt-BR = Estação de carregamento de bicicletas
    ro = Stație de încărcare pentru biciclete
    ru = Станция зарядки велосипедов
    sk = Nabíjacia stanica pre bicykle
    sv = Laddningsstation för cykel
    sw = Kituo cha Kuchaji Baiskeli
    th = สถานีชาร์จจักรยาน
    tr = Bisiklet Şarj İstasyonu
    uk = Велосипедна зарядна станція
    vi = Trạm sạc xe đạp
    zh-Hans = 自行车充电站
    zh-Hant = 自行車充電站
```

Reply:
It's correct for French.

Reply:
looks good for Hebrew

Reply:
PT and PT-BR good